package.json

```diff
@@ -63,6 +66,7 @@
   },
   "dependencies": {
     "@rollup/pluginutils": "^3 || ^4 || ^5",
+    "prettier": "^3.3.3",
```

Comment:
Prettier erroneously added here (should be in dev-deps, as you know).

Reply:
Also, I noticed you've removed `.commitlintrc.json` - It's no longer required?

Reply:
> Prettier erroneously added here (should be in dev-deps, as you know).

Ouch, my bad 😅

> Also, I noticed you've removed .commitlintrc.json - It's no longer required?

Still there, just changed to be a plain json rather than a js file